### ISSUE TITLE
'Model' object has no attribute 'hyp'

### ISSUE THREAD

**Issue by StuffyMonkey** [ultralytics/yolov5#8573]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I've imported yolov5n model and tried to use your loss function: `ultralytics/yolov5/utils/loss.py`. After passing model as argument I got the following error: 'Model' object has no attribute 'hyp'. Doesn't yolov5 from pytorch hub contain attribute 'hyp'? Moreover, i thought that hyperparams are not attributes only of the model, but also of optimizer (ex learning rate).
Full code:
`cnn = torch.hub.load('ultralytics/yolov5', 'yolov5n', autoshape=False, classes=1, force_reload=True)`
`loss = ComputeLoss(cnn)`

**Comment by glenn-jocher** [comment]
@StuffyMonkey model.hyp contains hyps needed for loss computation like balancing terms etc. These are attached here in train.py:
https://github.com/ultralytics/yolov5/blob/f8722b4429e80f96be04b36e4efd84ce6583bfa1/train.py#L284
